lib/spawn/cluster/state_handoff.ex

```diff
@@ -68,6 +68,20 @@ defmodule Spawn.Cluster.StateHandoff do
     {:reply, hosts, crdt_pid}
   end
 
+  def handle_call(:get_all_invocations, _from, crdt_pid) do
+    schedules =
+      crdt_pid
+      |> DeltaCrdt.to_map()
+      |> Map.values()
+      |> List.flatten()
+      |> Enum.map(& &1.opts[:invocations])
+      |> List.flatten()
+      |> Enum.reject(&is_nil/1)
+      |> Enum.uniq()
```

Comment:
This `Enum.uniq` is necessary because **maybe** the Actor is not reeeaally cleaning the actor hosts when leaving a node...

Reply:
Yes there is this bug as I reported above